lib/class-wp-theme-json.php

```diff
@@ -1064,6 +1065,18 @@ public function get_page_templates() {
 		}
 	}
 
+	/**
+	 * Returns the template part data of current theme.
+	 *
+	 * @return array
+	 */
+	public function get_template_part_data() {
```

Comment:
What if we rename this to `get_template_parts` so the getter mimics the field name?

```suggestion
	public function get_template_parts() {
```